README.md

```
@@ -1,28 +1,210 @@
+# E Buddy: AI Chatbot Development and Integration Platform
 
-## Team Members
-- Mohtasim-Hossain (Team Leader)
-- FahimSadikRashad
-- TanvirHossenShishir
+## Tyrocoders Team
 
-## Mentor
-- MinhazulHasan
+**Team Members:**
+- Mohtasim Hossain (Team Leader)
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines
null

(MD032, blanks-around-lists)